### DISCUSSION TITLE
Windows 11 ARM

### DISCUSSION THREAD

**Discussion by LckySndays** [leecher1337/ntvdmx64#287] [Q&A] [answered]
Will ntvdmx64 work on windows 11 arm?  anyone have tried it?

Seems like heavy modification might be needed to support arm https://github.com/leecher1337/ntvdmx64/issues/118

**Comment by leecher1337** [comment, marked as answer]
Nope, I don't see that MINNT build environment has ARM support and there are too many obstacles. I'm not aware of anyone wokring on that.

**Reply by WindowsNinetyTen** [reply]
I was wondering the same, but out of curiosity; I don't have an ARM Windows PC myself. And this answers the question for me as well. NTVDM _did_ get implemented as a full emulator in Alpha, DEC, MIPS and PowerPC versions of Windows NT, but it would most likely be much too difficult to rewrite any version for ARM.
